docs/upgrading_to_v3.0.md

```diff
@@ -0,0 +1,44 @@
+# Upgrading to v3.0
+
+The v3.0 release of *org-policy* is a backwards incompatible
+release.
+
+## Migration Instructions
+
+In previous versions of this module the resource hirerarchy node was not defined explicitly which caused problems when the module is combined with project/folder creation. The `policy_for` variable was introdused to address that issue.
```

Comment:
Typo: `s/introdused/introduced/`

Reply:
@ludoo thank you for being so attentive, that's fixed